nifi-commons/nifi-utils/src/main/java/org/apache/nifi/processor/util/StandardValidators.java

```diff
@@ -507,6 +507,20 @@ public ValidationResult validate(final String subject, final String input, final
     // FACTORY METHODS FOR VALIDATORS
     //
     //
+    public static Validator createAllowEmptyValidator(Validator delegate) {
```

Comment:
Please add some unit tests for this new validator.
You can find the existing tests in a different package for some reason: org.apache.nifi.util.validator.TestStandardValidators